Month2/Pointers_Exercise/Section1Pointers/Exercise7/Source7.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+int IncrementNumber(int NewIncreasedValue)
+{
+	return NewIncreasedValue +=1;
+}
```

Comment:
According to the exercise prompt, this was meant to be `void IncrementNumber(int* NumberPtr)` so it shouldn't return anything and should be passed a pointer as a parameter instead of an int.

Reply:
Updated